### PR TITLE
feat: add compass proto

### PR DESCRIPTION
**Pull request by mabdh**

### CODE REVIEW

File: odpf/compass/v1beta1/service.proto [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+  string type = 1 [(validate.rules).string = { in: [ "openended", "issues", "qanda", "all" ], ignore_empty: true }];
+  string state = 2 [(validate.rules).string = { in: [ "open", "closed", "all" ], ignore_empty: true }];
+  string owner = 3 [(validate.rules).string.ignore_empty = true];
+  string assignee = 4 [(validate.rules).string.ignore_empty = true];
+  string asset = 5 [(validate.rules).string.ignore_empty = true];
+  string labels = 6 [(validate.rules).string.ignore_empty = true];
+  string sort = 7 [(validate.rules).string = { in: [ "created_at", "updated_at" ], ignore_empty: true }];
+  string direction = 8 [(validate.rules).string = { in: [ "asc", "desc" ], ignore_empty: true  }];
+  uint32 size = 9 [(validate.rules).uint32 = { gte: 0, ignore_empty: true } ];
+  uint32 offset = 10 [(validate.rules).uint32 = { gte: 0, ignore_empty: true }];
```
Comment (rahmatrhd) [inline odpf/compass/v1beta1/service.proto:161]: how's this validation rules gonna be used?

Comment (mabdh) [reply odpf/compass/v1beta1/service.proto:161]: @rahmatrhd you can refer to this one https://github.com/envoyproxy/protoc-gen-validate#protoc-gen-validate-pgv

**Review by StewartJingga — APPROVED** [review]
LGTM

File: odpf/compass/v1beta1/service.proto (hunk 1 above)
Comment (mabdh) [reply odpf/compass/v1beta1/service.proto:161]: Like this one in columbus 	https://github.com/odpf/columbus/blob/f1b47eb31b512b492fad28b845c237b422a84fd3/api/v1beta1/discussion.go#L26

**Pull request merged**